drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -173,9 +173,16 @@ void ite_intc_irq_handler(const void *arg)
 uint8_t get_irq(void *arg)
 {
 	ARG_UNUSED(arg);
-	intc_irq = IVECT - IVECT_OFFSET_WITH_IRQ;
 
+	/* wait until two equal interrupt values are read */
```

Comment:
If I understand right, you have to wait for intc_irq to generate the interrupt number.  Can you add to this comment to explain the process?

Reply:
Added comment below.